### DISCUSSION TITLE
zero mAPs in the first 5 epochs

### DISCUSSION THREAD

**Discussion by sandylaker** [rwightman/efficientdet-pytorch#198] [Q&A]
Hi, 

I tried to train efficientdet_d1 on COCO dataset from scratch. Here are my settings:
* hardware: 8x A100 GPUs;
* batch size: 18 images on each GPU;
* lr: linearly scaled to 0.18 = (0.16 / 128) * 18 * 8.
* others: no amp, no fused momentum, 5 epochs for warm-up.

I also modified the train.py to enable monitoring the classification and regression losses. Although the losses kept decreasing in the first 5 epochs, the mAPs on validation set were near 0 all the time. Have you also observed this issue? Is it normal?

**Comment by rwightman** [comment]
@sandylaker If you have EMA enabled that's expected, and will likely remain like that for 30-100 epochs depending on the decay constant. If you do not have ema enabled then something is likely wrong.

**Reply by jaideep11061982** [reply]
I too face similar issue with eff det 2.
Loss dsnt comes down ,I.copied almost same dataset that worked last for me in kaggle gwd competition . Difference is only no of classes 
I hope I pass in  box cord in right format .
Ymin , x min , ymax x max , 
where x max is x +width , y max  is ymin +height 

Also Class loss plus box loss dsnt adds to total loss .
0.5... + .002 box loss ,total loss 0.7

Any help or pointers
